src/js/components/TableView.js

```diff
@@ -91,6 +95,16 @@ function TableRow({
   metadataRow,
 }) {
   const dayObs = window.APP_DATA.date.replaceAll("-", "")
+
+  // Entries in metadata keyed `"@{channel_name}"` will have their
+  // values show up in the table instead of a blank space.
+  const noEventReplacements = channels.reduce((obj, chan) => {
+    const chanReplace = metadataRow.hasOwnProperty("@" + chan.name)
+      ? metadataRow["@" + chan.name]
+      : null
```

Comment:
```suggestion
    const chanReplace = metadataRow["@" + chan.name] ?? null
```

Reply:
I missed it again! I really like that operator too.